common/common.scss

```diff
@@ -815,7 +816,11 @@ nav.post-controls {
 }
 
 section.post-menu-area {
-  padding: 10px;
+  padding: 10px 10px 10px 67px;

```

Comment:
That's an interesting number, `67px`... why do we need that?

Reply:
It is the width of the avatar. This is creating space to the left of the content to allow for the avatar to float without covering the cooked content.